src/args.rs

```diff
@@ -77,6 +77,24 @@ pub struct Args {
         help = "User agent to be used for sending requests"
     )]
     pub user_agent: String,
+
+    /// Regex filter to limit saving pages to only matched ones
+    #[structopt(
+        short,
+        long,
+        default_value = ".*",
+        help = "Regex filter to limit to only saving pages that match this expression"
+    )]
```

Comment:
Maybe it's better to try to convert to a regex now, cf Line 18 with the parse(...)

Reply:
✔️ Yes, this makes it a lot cleaner!